templates/details_base.html

```diff
@@ -92,6 +105,9 @@ <h2 class="govuk-heading-s govuk-!-margin-top-3">
           </ul>
           {% include "partial/esda_info.html" with is_esda=is_esda %}
         {% endblock metadata_list %}
+        <div class="govuk-body-s" style="text-align: right;">
```

Comment:
This will work, but I think it might be worth introducing a CSS class here for reusability.

We already have a custom class called `.summary-card`, so I think we should move that to `scss/components/_summary_card.scss` and add an additional class for this right align:

```scss
.app-summary-card {
    background-color: govuk-colour("light-grey");
    padding: govuk-spacing(3) govuk-spacing(4);
    margin-bottom: govuk-spacing(6);
    overflow-wrap: break-word;
}

.app-summary-card .app-summary-card__footer  {
  text-align: right;
}
```

the above would bring it in line with [BEM naming convention](https://css-tricks.com/bem-101/) we are using elsewhere and the `app-` prefix for custom components.